teamA/lib/Controller/main_controller.dart

```diff
@@ -49,49 +47,17 @@ class MainController
     isLoggedIn = false;
   }
 
-  Future<bool> updateCourses() async 
-  {
-    var moodleApi = MoodleApiSingleton();
-    try {
-      // courses = await moodleApi.getCourses();
-      courses = await moodleApi.getUserCourses();
-      if (courses.isNotEmpty) {
-        courses.removeAt(
-            0); // first course is always "Moodle" - no need to show it
-      }
-      return true;
-    } catch (e) {
-      if (kDebugMode) {
-        print(e);
-      }
-      courses = [];
-      return false;
-    }
-  }
-
   Future<bool> isUserLoggedIn() async 
   {
     return isLoggedIn;
   }
 
-  Future<bool> selectCourse(int index) async{
-    if (index < courses.length){
-      selectedCourse = courses[index];
+  bool selectCourse(int index) {
```

Comment:
For boolean functions it's typically better to name them as if they were describing the thing they're testing. A more appropriate name would be something like isCourseSelected or canCourseBeSelected something along those lines.

Reply:
It just makes it immediately clear what the function is doing since seeing 

if (selectCourse(0)) { doesn't show exactly what it's testing at first glance

Reply:
It's a holdover from when I used it directly in an if statement, and was based off of the login function. Changed the return type to void, another function can be added if needed.